### DISCUSSION TITLE
I followed the tutorial on YouTube but did not get the results

### DISCUSSION THREAD

**Discussion by yikaizhou252** [Deerrorer/dotfiles3#6] [Q&A] [answered]
Apologies as I'm new to Arch. So I have a freshly installed Archlinux with the archinstall script running KDE Plasma and I pulled and ran the script as shown in the video. The only noticeable difference for me was the login screen but the windows and displays stayed pretty much the same as the vanilla version of Plasma. 

Am I supposed to install the Linux version without GUI as you did? Please let me know and thanks.

**Comment by Keyitdev** [comment, marked as answer]
You need to change the session in sddm. In the login screen, click (sometimes multiple times) Session: KDE Plasma. And switch it to i3.
